library/src/main/assets/google_map.html

```diff
@@ -106,12 +106,13 @@
   });
 }
 
-function addMarkerWithId(lat, lng, id, title, snippet) {
+function addMarkerWithId(lat, lng, id, title, snippet, draggable) {
```

Comment:
Could you create a new method that includes the additional parameter instead of changing this method's signature as to not break usages of the old method?